tests/flow/test_concurrent_query.py

```diff
@@ -127,43 +112,70 @@ def test_05_concurrent_read_delete(self):
         # Delete graph via Redis DEL key.
         ##############################################################################################
         self.populate_graph()
+        pool = Pool(nodes=CLIENT_COUNT)
 
         q = """UNWIND (range(0, 10000)) AS x WITH x AS x WHERE (x / 900) = 1 RETURN x"""
         queries = [q] * CLIENT_COUNT
-        results = run_concurrent(queries, thread_run_query)
+        # invoke queries
+        m = pool.amap(thread_run_query, queries)
+
+        self.conn.delete(GRAPH_ID)
+
+        # wait for processes to return
+        m.wait()
+
+        # get the results
+        results = m.get()
 
         # validate result.
-        self.env.assertTrue(all([r.result_set[0][0] == 900 for r in results]))
+        self.env.assertTrue(all([r["result_set"][0][0] == 900 for r in results]))
 
         # Make sure Graph is empty, e.g. graph was deleted.
         resultset = self.graph.query("MATCH (n) RETURN count(n)").result_set
         self.env.assertEquals(resultset[0][0], 0)
-
         ##############################################################################################        
         # Delete graph via Redis FLUSHALL.
         ##############################################################################################
         self.populate_graph()
         q = """UNWIND (range(0, 10000)) AS x WITH x AS x WHERE (x / 900) = 1 RETURN x"""
         queries = [q] * CLIENT_COUNT
+        # invoke queries
+        m = pool.amap(thread_run_query, queries)
+
+        self.conn.flushall()
+
+        # wait for processes to return
+        m.wait()
+
+        # get the results
+        results = m.get()
```

Comment:
do we want to validate `results ` ?

Reply:
sorry don't need run_concurrent here so there is validation